FifthLesson/VectorProgram/VectorProgram/Model/Vector.cs

```diff
@@ -1,115 +1,96 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using VectorProgram.Controller;
 
 namespace VectorProgram.Model
 {
-    public sealed class Vector
+    public class Vector : IEquatable<Vector>
     {
-        public double CoordinateFirst { get; private set; }
-        public double CoordinateSecond { get; private set; }
-        public double CoordinateThird { get; private set; }
-
+        public double FirstCoordinate { get; private set; }
+        public double SecondCoordinate { get; private set; }
+        public double ThirdCoordinate { get; private set; }
         public double Length
         {
-            get
-            {
-                return Math.Round(Math.Sqrt(CoordinateFirst * CoordinateFirst + CoordinateSecond * CoordinateSecond + CoordinateThird * CoordinateThird), 3);
-            }
+            get => Math.Round(Math.Sqrt(Math.Pow(FirstCoordinate, 2) + Math.Pow(SecondCoordinate, 2) + Math.Pow(ThirdCoordinate, 2)), 3);
         }
 
-        public Vector()
+        public Vector() { }
+
+        public Vector(double coordinateFirst, double coordinateSecond, double coordinateThird)
```

Comment:
Please, rename **coordinateFirst** to **firstCoordinate**. Do the same with other parameters

Reply:
Corrected.